### PR TITLE
Extract Password from variable initial_root_password

### PR DESCRIPTION
**Pull request by buluma**
Feature request 1

### CODE REVIEW

File: molecule/default/converge.yml [hunk 1]
```diff
@@ -10,3 +10,23 @@
       gitlab_cleanup_ruby: no
       gitlab_trusted_certs:
         - isrgrootx1.pem  # A root certificate for letsencrypt.
+  post_tasks:
```
Comment (buluma) [inline molecule/default/converge.yml:13]: #21

**Review by buluma — COMMENTED** [review]
@patsevanton should be doable with gitlab_initial_root_password var

**Comment by patsevanton** [comment]
Don`t show task in log on latest version role:
```
TASK [buluma.gitlab : place configuration templated] ***********************************************************************************************************************************
Monday 28 March 2022  10:54:54 +0600 (0:00:00.015)       0:06:09.288 **********
changed: [gitlab]

TASK [buluma.gitlab : make trusted-certs directories] **********************************************************************************************************************************
Monday 28 March 2022  10:54:56 +0600 (0:00:02.470)       0:06:11.759 **********
skipping: [gitlab]

TASK [buluma.gitlab : place trusted certificates] **************************************************************************************************************************************
Monday 28 March 2022  10:54:56 +0600 (0:00:00.013)       0:06:11.773 **********
skipping: [gitlab]

TASK [buluma.gitlab : place configuration static] **************************************************************************************************************************************
Monday 28 March 2022  10:54:56 +0600 (0:00:00.013)       0:06:11.786 **********
skipping: [gitlab]

RUNNING HANDLER [buluma.gitlab : run gitlab-ctl reconfigure] ***************************************************************************************************************************
Monday 28 March 2022  10:54:56 +0600 (0:00:00.016)       0:06:11.803 **********
changed: [gitlab]

RUNNING HANDLER [buluma.gitlab : uninstall ruby] ***************************************************************************************************************************************
Monday 28 March 2022  10:59:25 +0600 (0:04:28.112)       0:10:39.915 **********
changed: [gitlab]

PLAY RECAP *****************************************************************************************************************************************************************************
gitlab                     : ok=67   changed=8    unreachable=0    failed=0    skipped=45   rescued=0    ignored=0

Monday 28 March 2022  10:59:38 +0600 (0:00:13.382)       0:10:53.298 **********
===============================================================================
buluma.gitlab : install gitlab ------------------------------------------------------------------------------------------------------------------------------------------------ 316.62s
buluma.gitlab : run gitlab-ctl reconfigure ------------------------------------------------------------------------------------------------------------------------------------ 268.11s
buluma.gitlab : install requirements ------------------------------------------------------------------------------------------------------------------------------------------- 24.78s
buluma.gitlab : uninstall ruby ------------------------------------------------------------------------------------------------------------------------------------------------- 13.38s
buluma.gitlab : install apt repository ----------------------------------------------------------------------------------------------------------------------------------------- 12.31s
buluma.gitlab : install ruby to allow syntax check of gitlab.rb ----------------------------------------------------------------------------------------------------------------- 7.62s
Gathering Facts ----------------------------------------------------------------------------------------------------------------------------------------------------------------- 4.28s
buluma.gitlab : place configuration templated ----------------------------------------------------------------------------------------------------------------------------------- 2.47s
buluma.gitlab : install apt repository key -------------------------------------------------------------------------------------------------------------------------------------- 1.82s
buluma.gitlab : test if item in gitlab_rails_ldap_servers is set correctly ------------------------------------------------------------------------------------------------------ 0.06s
buluma.gitlab : test if group_base item in gitlab_rails_ldap_servers is set correctly ------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_default_projects_feature_* are set correctly ----------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if admin_group item in gitlab_rails_ldap_servers is set correctly ------------------------------------------------------------------------------------------ 0.02s
buluma.gitlab : test if gitlab_letsencrypt_contact_emails is set correctly ------------------------------------------------------------------------------------------------------ 0.02s
buluma.gitlab : test if gitlab_rails_db_encoding is set correctly --------------------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_letsencrypt_auto_renew_day_of_month is set correctly --------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_rails_db_prepared_statements is set correctly ---------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_letsencrypt_auto_renew_minute is set correctly --------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_rails_gitlab_email_smime_enabled is set correctly ------------------------------------------------------------------------------------------------ 0.02s
buluma.gitlab : test if gitlab_rails_smtp_ca_path is set correctly -------------------------------------------------------------------------------------------------------------- 0.02s
###############
Execution time was 712 s.
Duration: 0 hours 11 minutes 52 seconds
###############
```

**Comment by patsevanton** [comment]
Don`t show task "Extract Password from variable initial_root_password" in log on latest version role:


If use inventory
```
all:
  children:
    gitlab:
      hosts:
        "gitlab":
          ansible_host: "51.250.97.225"
  vars:
    gitlab_external_url: "https://gitlab.51.250.97.225.sslip.io"
    ansible_user:  ubuntu
    ansible_ssh_private_key_file: ~/.ssh/id_rsa
    gitlab_letsencrypt: yes
    gitlab_letsencrypt_contact_emails:
      - you.email@gmail.com
    gitlab_letsencrypt_group: root
    gitlab_letsencrypt_key_size: 2048
    gitlab_letsencrypt_owner: root
    gitlab_letsencrypt_wwwroot: /var/opt/gitlab/nginx/www
    gitlab_letsencrypt_auto_renew: yes
    gitlab_letsencrypt_auto_renew_hour: 0
    gitlab_letsencrypt_auto_renew_minute: nil
    gitlab_letsencrypt_auto_renew_day_of_month: 7
    gitlab_letsencrypy_auto_renew_log_directory: /var/log/gitlab/lets-encrypt
    gitlab_initial_root_password: true
```

Playbook
```
- hosts: gitlab
  become: yes
  roles:
    - role: buluma.gitlab
```

Output
```
TASK [buluma.gitlab : place configuration static] **************************************************************************************************************************************
Monday 28 March 2022  12:04:07 +0600 (0:00:00.013)       0:06:12.160 **********
skipping: [gitlab]

RUNNING HANDLER [buluma.gitlab : run gitlab-ctl reconfigure] ***************************************************************************************************************************
Monday 28 March 2022  12:04:07 +0600 (0:00:00.016)       0:06:12.177 **********
changed: [gitlab]

RUNNING HANDLER [buluma.gitlab : uninstall ruby] ***************************************************************************************************************************************
Monday 28 March 2022  12:08:52 +0600 (0:04:44.438)       0:10:56.615 **********
changed: [gitlab]

PLAY RECAP *****************************************************************************************************************************************************************************
gitlab                     : ok=67   changed=8    unreachable=0    failed=0    skipped=45   rescued=0    ignored=0

Monday 28 March 2022  12:09:07 +0600 (0:00:14.739)       0:11:11.354 **********
===============================================================================
buluma.gitlab : install gitlab ------------------------------------------------------------------------------------------------------------------------------------------------ 311.87s
buluma.gitlab : run gitlab-ctl reconfigure ------------------------------------------------------------------------------------------------------------------------------------ 284.44s
buluma.gitlab : install requirements ------------------------------------------------------------------------------------------------------------------------------------------- 27.67s
buluma.gitlab : uninstall ruby ------------------------------------------------------------------------------------------------------------------------------------------------- 14.74s
buluma.gitlab : install apt repository ----------------------------------------------------------------------------------------------------------------------------------------- 13.17s
buluma.gitlab : install ruby to allow syntax check of gitlab.rb ----------------------------------------------------------------------------------------------------------------- 7.89s
Gathering Facts ----------------------------------------------------------------------------------------------------------------------------------------------------------------- 4.72s
buluma.gitlab : place configuration templated ----------------------------------------------------------------------------------------------------------------------------------- 2.89s
buluma.gitlab : install apt repository key -------------------------------------------------------------------------------------------------------------------------------------- 2.07s
buluma.gitlab : test if item in gitlab_rails_ldap_servers is set correctly ------------------------------------------------------------------------------------------------------ 0.06s
buluma.gitlab : test if gitlab_default_projects_feature_* are set correctly ----------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if group_base item in gitlab_rails_ldap_servers is set correctly ------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if sync_ssh_keys item in gitlab_rails_ldap_servers is set correctly ---------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if admin_group item in gitlab_rails_ldap_servers is set correctly ------------------------------------------------------------------------------------------ 0.02s
buluma.gitlab : test if gitlab_letsencrypt_auto_renew is set correctly ---------------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_rails_smtp_password is set correctly ------------------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_rails_gitlab_email_reply_to is set correctly ----------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_rails_db_host is set correctly ------------------------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_rails_smtp_authentication is set correctly ------------------------------------------------------------------------------------------------------- 0.02s
buluma.gitlab : test if gitlab_cleanup_ruby is set correctly -------------------------------------------------------------------------------------------------------------------- 0.02s
###############

```

**Comment by buluma** [comment]
let me have a look, I will update.

**Pull request merged**